pulpcore/cli/common/generic.py

```diff
@@ -457,6 +457,22 @@ def _multi_option_callback(
         kwargs["callback"] = _multi_option_callback
     else:
         kwargs["callback"] = _option_callback
+
+    if "help" not in kwargs:
+        kwargs["help"] = _(
+            "Referenced resource, in the form {plugin_form}{type_form}<name> or by href. "
+            "{plugin_default}{type_default}"
+        ).format(
+            plugin_form=_("[<plugin>:]") if default_plugin else _("<plugin>"),
+            type_form=_("[<resource_type>:]") if default_type else _("<resource_type>"),
```

Comment:
I believe we still need the colons:
```suggestion
            plugin_form=_("[<plugin>:]") if default_plugin else _("<plugin>:"),
            type_form=_("[<resource_type>:]") if default_type else _("<resource_type>:"),
```

Reply:
Good catch - I have updated the commit.